papas/pfobjects.py

```diff
@@ -18,12 +22,16 @@ class PFObject(object):
     '''
 
 
-    def __init__(self, pfobjecttype=Identifier.PFOBJECTTYPE.NONE, subtype='u'):
+    def __init__(self, pfobjecttype=Identifier.PFOBJECTTYPE.NONE, subtype='u', identifiervalue = 0.0):
+        '''@param pfobjecttype: type of the object to be created (used in Identifier class) eg Identifier.PFOBJECTTYPE.ECALCLUSTER
```

Comment:
Thank you for using the epydoc tags here.
Could you please adopt the epydoc notation for all docstrings? 
do you know how to generate the documentation? if not I should explain you, let me know.